brave-spark-framework-filter/src/main/java/com/github/kristofa/brave/spark/BraveSparkExceptionHandler.java

```diff
@@ -0,0 +1,49 @@
+package com.github.kristofa.brave.spark;
+
+import com.github.kristofa.brave.Brave;
+import com.github.kristofa.brave.ServerTracer;
+import spark.ExceptionHandler;
+import spark.Request;
+import spark.Response;
+import spark.utils.StringUtils;
+import zipkin.Constants;
+
+import static com.github.kristofa.brave.internal.Util.checkNotNull;
+
+public final class BraveSparkExceptionHandler implements ExceptionHandler {
+
+    private final Brave brave;
+
+    private final ExceptionHandler exceptionHandler;

```

Comment:
The same thought as you,but I thing change brave-spark-framework-filter  to sparkjava better.